### DISCUSSION TITLE
deleted

### DISCUSSION THREAD

**Discussion by RimuruChan** [vfsfitvnm/frida-il2cpp-bridge#210] [Q&A]

**Comment by RimuruChan** [comment]


**Reply by esauvisky** [reply]
Are they intermittent? Do they break some game mechanic? I've gone through similar ones, with some progress but not definite success.

Give me a shout on discord or tg and we might be able to work on it together until we find some solution :)

**Reply by ChaomengOrion** [reply]
> Other hooks are working, but this one causes some exceptions in Unity. Does anyone have any ideas?

This is not a hook problem. You can know it through decompilation.
And can you provide a social account to let me tell you the solution(like QQ).

**Reply by esauvisky** [reply]
Yes, I'm aware, this is not an issue with the instrumentation framework nor this library itself. If this is the same behavior (it looks like) I get at another completely different game, then it's a race condition caused by the instrumentation of asynchronous coroutines that invoke delegates at "the wrong time" after object instances were destroyed or during they are being kept locked because of thread unsafe operations. 

Add me at discord (`@emi~#0001`) and we might be able to help each other. Unfortunately the maintainer of this repository has nothing to do with it, it's not a problem with frida-il2cpp-bridge
